packages/theme/css/foundations/color-next.css

```diff
@@ -0,0 +1,161 @@
+.salt-theme-next.salt-theme {
+  --salt-color-aqua-400: rgb(111, 219, 222);
+  --salt-color-aqua-500: rgb(42, 130, 133);
+  --salt-color-aqua-600: rgb(21, 92, 94);
+  --salt-color-asphalt: rgb(24, 25, 26);
+  --salt-color-asphalt-40a: rgba(24, 25, 26, 0.4);
+  --salt-color-autumn-400: rgb(242, 159, 99);
+  --salt-color-autumn-500: rgb(189, 91, 21);
+  --salt-color-autumn-600: rgb(128, 58, 8);
+  --salt-color-black: rgb(0, 0, 0);
+  --salt-color-black-15a: rgba(0, 0, 0, 0.15);
+  --salt-color-black-25a: rgba(0, 0, 0, 0.25);
+  --salt-color-black-40a: rgba(0, 0, 0, 0.4);
+  --salt-color-black-70a: rgba(0, 0, 0, 0.7);
+  --salt-color-black-8a: rgba(0, 0, 0, 0.08);
+  --salt-color-blue-100: rgb(202, 231, 252);
+  --salt-color-blue-200: rgb(167, 203, 234);
+  --salt-color-blue-200-40a: rgba(167, 203, 234, 0.4);
+  --salt-color-blue-300: rgb(131, 176, 216);
+  --salt-color-blue-300-40a: rgba(131, 176, 216, 0.4);
+  --salt-color-blue-400: rgb(92, 149, 199);
+  --salt-color-blue-400-40a: rgba(92, 149, 199, 0.4);
+  --salt-color-blue-500: rgb(43, 124, 181);
+  --salt-color-blue-500-40a: rgba(43, 124, 181, 0.4);
```

Comment:
it would be probably be more tokens, esp given the new colours, but i always liked this approach https://github.com/jpmorganchase/salt-ds/discussions/2461 that ensures you are definitely using the same r,g,b values without having to repeat